tests/testthat/test-geo_topo.R

```diff
@@ -42,3 +42,53 @@ test_that("topo2geo works", {
   expect_s3_class(w, "geofeaturecollection")
 })
 
+
+
+
+test_that("quantization=0 in geo2topo", {
```

Comment:
For the tests with a default of the `quantization` argument, can you fit them into existing tests (lines 22-43) so that we're not running the same code more than once?